core/lib/Thelia/Module/BaseModuleInterface.php

```diff
@@ -26,5 +26,7 @@ public function preDeactivation(ConnectionInterface $con = null);
 
     public function postDeactivation(ConnectionInterface $con = null);
 
+    public function update($currentVersion, $newVersion, ConnectionInterface $con = null);
```

Comment:
What is the goal of that method ?
And what are ```$currentVersion``` and ```$newVersion``` ? Are they Thelia or Module version ? :)